src/cloudShellRunner.ts

```diff
@@ -52,50 +52,67 @@ export class CloudShellRunner extends BaseRunner {
         const promptMsg = 'Run ansible playbook in Cloudshell will generate Azure usage fee since need uploading playbook to CloudShell !';
-        vscode.window.showWarningMessage(promptMsg, msgOption, msgItem, cancelItem).then(
-            response => {
-                if (response === msgItem) {
-                    const accountApi: AzureAccount = vscode.extensions.getExtension<AzureAccount>("ms-vscode.azure-account")!.exports;
-
-                    openCloudConsole(accountApi, OSes.Linux, [playbook], this._outputChannel, tempFile).then(terminal => {
-                        var count = 30;
-                        if (terminal) {
-                            var _localthis = this;
-                            var interval = setInterval(function () {
-                                count--;
-                                if (count > 0) {
-                                    if (fsExtra.existsSync(tempFile)) {
-                                        fsExtra.removeSync(tempFile);
-                                        if (utilities.isTelemetryEnabled()) {
-                                            terminal.sendText('export ' + Constants.UserAgentName + '=' + utilities.getUserAgent());
-                                        }
-                                        terminal.sendText('ansible-playbook ' + path.basename(playbook));
-                                        terminal.show();
-
-                                        count = 0;
-                                    }
-                                } else {
-                                    _localthis.stop(interval);
+
+        this.showPrompt().then(() => {
+            const accountApi: AzureAccount = vscode.extensions.getExtension<AzureAccount>("ms-vscode.azure-account")!.exports;
+
+            openCloudConsole(accountApi, OSes.Linux, [playbook], this._outputChannel, tempFile).then(terminal => {
+                var count = 30;
+                if (terminal) {
+                    var _localthis = this;
+                    var interval = setInterval(function () {
+                        count--;
+                        if (count > 0) {
+                            if (fsExtra.existsSync(tempFile)) {
+                                fsExtra.removeSync(tempFile);
+                                if (utilities.isTelemetryEnabled()) {
+                                    terminal.sendText('export ' + Constants.UserAgentName + '=' + utilities.getUserAgent());
                                 }
-                            }, 500);
+                                terminal.sendText('ansible-playbook ' + path.basename(playbook));
+                                terminal.show();
+
+                                count = 0;
+                            }
                         } else {
-                            this._outputChannel.appendLine('\nConnecting to cloud shell failed, please retry.');
```

Comment:
Let's just be consistend with cloud shell naming. Sometimes it's "cloud shell", sometimes "Cloudshell".
Maybe we should be consistent with this doc: https://docs.microsoft.com/en-us/azure/cloud-shell/overview
and just have "Cloud Shell" everywhere

Reply:
nice. fixed